Patch borrowed from OpenEmbedded, available at
recipes/libaio/libaio-0.3.106/destdir.patch in their source tree.

It just adds support for the traditional DESTDIR variable to install
the library in a different sysroot than the normal /.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: libaio-0.3.109/Makefile
===================================================================
--- libaio-0.3.109.orig/Makefile
+++ libaio-0.3.109/Makefile
@@ -23,7 +23,7 @@
 	@$(MAKE) -C src
 
 install:
-	@$(MAKE) -C src install prefix=$(prefix) includedir=$(includedir) libdir=$(libdir)
+	@$(MAKE) -C src install prefix=$(DESTDIR)$(prefix) includedir=$(DESTDIR)$(includedir) libdir=$(DESTDIR)$(libdir)
 
 check:
 	@$(MAKE) -C harness check
